recipes/magpylib/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "magpylib" %}
+{% set version = "1.0.2b0" %}
+{% set pypiSourceChecksum = 'fde2f8641a265461a9eb50259c5fccdf82822b57a441e3eae7a99f2861f3022e' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ pypiSourceChecksum }}
```

Putting the checksum here will make it easier for the update bot:
```suggestion
  sha256: fde2f8641a265461a9eb50259c5fccdf82822b57a441e3eae7a99f2861f3022e
```